egs/wsj/s5/steps/nnet3/compute_and_adjust_priors.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env python
+
+# Copyright 2016    Pegah Ghahremani.
+# Apache 2.0.
+
+""" This script used to compute average posterior and re-adjust prior
```

Comment:
I will remove the ali_dir option, but will keep the script, it is like python version of adjust_priors.sh.